components/header/header.styles.ts

```diff
@@ -0,0 +1,103 @@
+import { css } from 'lit';
+import { cva } from "class-variance-authority";
+
+export const headerVariants = cva(
```

Comment:
I do feel at some point we should re-evaluate how we're doing styling, these files do seem a little hacky, perhaps we could look at https://panda-css.com/ or something similar in the future.

Reply:
The current approach is pretty similar of what Shoelace uses, for example:

https://github.com/shoelace-style/shoelace/blob/next/src/components/button/button.styles.ts

Don't you think Panda CSS would introduce more confusion and more code to maintain? currently we're using common CSS, with global variables defined by Shoelace and some extras by us.

Reply:
I do quite like the simplicity of just CSS actually 👍

But let's sit and reevaluate at some point in the future to see if there is any refinement that could be done

Reply:
For sure, just something to think about and evaluate in the future! 👍